boilerplate/flyte/golang_test_targets/download_tooling.sh

```diff
@@ -35,6 +36,22 @@ pushd "$tmp_dir"
 for tool in "${tools[@]}"; do
 	echo "Installing ${tool}"
 	GO111MODULE=on go install $tool
+	# If tool is our mockery fork, we need to rename the binary to mockery-fork
+	if [[ $tool == "github.com/EngHabu/mockery/cmd/mockery" ]]; then
+		echo "Renaming mockery to mockery-fork"
+		mv $(go env GOPATH)/bin/mockery $(go env GOPATH)/bin/mockery-fork
+	fi
+	# If tool is named vektra/mockery/v2, we need to rename the binary to mockery-v2
+	if [[ $tool == "github.com/vektra/mockery/v2@v2.40.3" ]]; then
+		echo "Renaming mockery to mockery-v2"
+		mv $(go env GOPATH)/bin/mockery $(go env GOPATH)/bin/mockery-v2
```

Comment:
Depending on the timeline to manage the migration, do you know of a linter that would let us block new usage of `go:generate mockery`? I was looking at [forbidigo](https://golangci-lint.run/usage/linters/#forbidigo), but I don't see a file exclusion

Reply:
Good idea, although in the case of `forbidigo` it looks like it can't handle comments. In any case, I'm going to experiment with this in a separate PR. Maybe there's a way to restrict the linter to files in a PR (this might involve running a command via a `run` block in a gh action to list the go files in a PR, something like `git diff --name-only master | grep '.go$'` and pass that list to this tool).